cmd/pctl/install.go

```diff
@@ -43,12 +42,6 @@ func installCmd() *cli.Command {
 				Value: "",
 				Usage: "The name of the ConfigMap which contains values for this profile.",
 			},
-			&cli.StringFlag{
```

Comment:
We should keep the ability to specify a location though.

Reply:
yeah thats fair. I `rm`'d this since its now a folder rather than a file. I'll add in a follow on PR. Keen to keep this as slim as possible :smile: